mne/tests/test_report.py

```diff
@@ -167,7 +168,16 @@ def test_report_raw_psd_and_date():
 
     # DATE_NONE functionality
     report = Report()
-    raw.anonymize()
+
+    # old style (pre 0.20) date anonymization
+    raw.info['meas_date'] = None
+    for key in ('file_id', 'meas_id'):
+        value = raw.info.get(key)
+        if value is not None:
+            assert 'msecs' not in value
+            value['secs'] = DATE_NONE[0]
+            value['usecs'] = DATE_NONE[1]
```

Comment:
I would have kept the call to anonymize and updated the test. Is this doable ?

Reply:
This test is to ensure the report will correctly handle data with DATE_NONE as the meas_date. calling anonymize doesn't do that anymore so I had to put the old style anonymize back into the test. We presumably still have alot of data with DATE_NONE out in the wild so keeping this seems reasonable.

I did add a smoke test using the new anonymize, so we won't add something to anonymize in the future that breaks basic report rendering.